pkg/clusteroperator/list.go

```diff
@@ -131,3 +131,50 @@ func WaitForAllClusteroperatorsStopProgressing(
 
 	return false, err
 }
```

Comment:
Please change commit message to:
clusteroperator: Added VerifyClusterOperatorsVersion func

Reply:
done, thanks